src/components/annotationSearch/index.jsx

```diff
@@ -16,7 +19,7 @@ const lists = {
   organize: ['AND', 'OR', '(', ')'],
   paranthesis: ['(', ')'],
   condition: ['AND', 'OR'],
-  type: ['modality', 'diagnosis', 'anatomy'],
+  type: ['modality', 'observation', 'anatomy'],
```

Comment:
did you change the item in the dropdown to observation also?

Reply:
Since All UI elements look up these arrays for terms, they are automatically updated.